### PR TITLE
Tensorflow lite

### PR DESCRIPTION
**Pull request by antoinemrcr**
change models from coreml to tensorflow lite

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,28 +83,33 @@ You will need to download our pre-trained models to build the demo application.
...
 
 ```shell
 # If you are in sense
-mv ./resources/coreml/sensenet.mlmodel ../sense-iOS/sense-iOS/sensenet.mlmodel
+mv ./resources/model_conversion/model.tflite ../sense-iOS/sense-iOS/model.tflite
 ```
 
 #### Step 3: Build the project
-
+Before Building the project, you need to install the tensorflow lite pods.
```
Comment (sunny-panchal) [inline README.md:107]: Should move this to the requirements section of the readme and add this:

```
brew install cocoapods
```

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,24 @@
 .DS_Store
-*/sensenet.mlmodel
+*/model.tflite
+# Xcode
+#
```
Comment (sunny-panchal) [inline .gitignore:4]:
```suggestion

# Xcode
```

File: Podfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Uncomment the next line to define a global platform for your project
```
Comment (sunny-panchal) [inline Podfile:1]: Asking to uncomment a line that's already uncommented

File: README.md [hunk 4]
```diff
@@ -83,28 +83,33 @@ You will need to download our pre-trained models to build the demo application.
 
 This section will explain how you can deploy our pre-trained models, or your own custom model, to an iOS application. 
 
-#### Step 1: Converting a Pytorch model to CoreML
+#### Step 1: Converting a Pytorch model to Tensorflow-Lite
 
 The iOS demo requires a CoreML version of our model checkpoint which you can produce using the script provided in
```
Comment (sunny-panchal) [inline README.md:88]:
```suggestion
The iOS demo requires a Tensorflow Lite version of our model checkpoint which you can produce using the script provided in
```

File: sense-iOS/InferenceLocal.swift [hunk 5]
```diff
@@ -1,99 +1,278 @@
 import Foundation
-import CoreML
 import Accelerate
 import AVFoundation
+import TensorFlowLite
```
Comment (sunny-panchal) [inline sense-iOS/InferenceLocal.swift:4]: not sorted

File: README.md [hunk 6]
```diff
@@ -83,28 +83,33 @@ You will need to download our pre-trained models to build the demo application.
...
  `sense` which, for our pre-trained gesture control model, can be run using:
 
 ```shell
-python scripts/conversion/convert_to_coreml.py --backbone=efficientnet --classifier=efficient_net_gesture_control --output_name=sensenet
+python tools/conversion/convert_to_tflite.py --backbone=efficientnet --classifier=efficient_net_gesture_control --output_name=model
 ```
 
-You should now have the following CoreML file: `sense/resources/coreml/sensenet.mlmodel`.
+You should now have the following CoreML file: `sense/resources/model_conversion/model.tflite`.
```
Comment (sunny-panchal) [inline README.md:95]:
```suggestion
You should now have the following Tensorflow Lite file: `sense/resources/model_conversion/model.tflite`.
```

**Review by sunny-panchal — COMMENTED** [review]
Just some minor comments -- someone else should take a look at the iOS files

File: sense-iOS/InferenceLocal.swift (hunk 5 above)
Comment (antoinemrcr) [reply sense-iOS/InferenceLocal.swift:4]: And ?

**Comment by florianletsch** [comment]
General question: Are we completely abandoning CoreML in favor of TFlite? There's no drawback?

**Comment by antoinemrcr** [comment]
> General question: Are we completely abandoning CoreML in favor of TFlite? There's no drawback?

It is a bit more computationally intensive on devices that don't have a neural engine (up to the iphone 8 / iphone X generation).
On my tests, it was still running on the iphone 8. The advantages are severals: only one model for android and iOS, less need to do modification on the code. It might also be possible to run mobilenet and efficientnet backbone without code change

File: README.md [hunk 7]
```diff
@@ -77,34 +77,46 @@ Next, follow the instructions for [sense](https://github.com/TwentyBN/sense) to
...
+```
+
 --- 
 
 ## Getting Started
 
 This section will explain how you can deploy our pre-trained models, or your own custom model, to an iOS application. 
 
-#### Step 1: Converting a Pytorch model to CoreML
+#### Step 1: Converting a Pytorch model to Tensorflow-Lite
```
Comment (guillaumebrg) [inline README.md:99]:
```suggestion
#### Step 1: Converting a Pytorch model to Tensorflow Lite
```

File: README.md [hunk 8]
```diff
@@ -77,34 +77,46 @@ Next, follow the instructions for [sense](https://github.com/TwentyBN/sense) to
 You will need to download our pre-trained models to build the demo application. Once again, please follow the
  instructions in [sense](https://github.com/TwentyBN/sense) to access them (you will have to create an account and agree to our terms and conditions).
 
+#### Step 4: Install the pods
+
+This project relies on Pods to install Tensorflow Lite.
+If you don't have cocoapods install on your mac, you can install it using brew:
```
Comment (guillaumebrg) [inline README.md:83]:
```suggestion
If you don't have `cocoapods` installed on your mac, you can install it using brew:
```

**Review by guillaumebrg — APPROVED** [review]

**Pull request merged**